src/Pages/AboutUsManager/AboutUsManager.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect} from 'react';
+import {
+  getAllOfficers,
+  createOfficer,
+  editOfficer,
+  deleteOfficer
+} from '../../APIFunctions/OfficerManager';
+import AboutUsCard from '../AboutUs/AboutUsCard.js';
+import AboutUsCircle from '../AboutUs/AboutUsCircle.js';
+import { officerModalState } from '../../Enums';
+import AboutUsManagerModal from './AboutUsManagerModal.js';
+import { Button } from 'reactstrap';
+import Header from '../../Components/Header/Header';
+import './AboutUsManager.css';
+export default function AboutUsManager(props){
+  const [modal, setModal] = useState(false);
+  const [officer, setOfficer] = useState();
+  const [modalState, setModalState] = useState(officerModalState.SUBMIT);
+  const [officers, setOfficers] = useState([]);
+  const headerProps = {
+    title: 'About Us Manager'
+  };
+
+
+  async function populateOfficers(){
+    const officerData = await getAllOfficers();
+    if(!officerData.error) setOfficers(officerData.responseData);
+  }
+
+  useEffect(() => {
+    populateOfficers();
+  }, []);
+
+  function toggle(){
+    setModal(!modal);
+  }
+
+  function toggleEditOfficer(officer) {
+    setModalState(officerModalState.EDIT);
+    setOfficer(officer);
+    toggle();
+  }
+
+  function toggleCreateOfficer(officer) {
```

Comment:
```suggestion
  function toggleCreateOfficer() {
```

I don't think you use `officer`?